package.json

```diff
@@ -10,7 +10,7 @@
     "test": "echo \"SSSSHhhhhh... no test specified\"",
     "dev:tsc": "tsc",
     "clean": "rimraf dist/ *.tgz",
-    "prepare": "npm run clean && npm run dev:tsc"
+    "prepare": "npm run clean && npm run dev:tsc && chmod +x ./dist/index.js"
```

Comment:
Won't work on any non-*nix based machines :) shouldn't be required for publishing and locally you can just use `node dist/index.js`